enterprise/app/workflows/workflows.tsx

```diff
@@ -211,6 +211,18 @@ class ListWorkflowsComponent extends React.Component<ListWorkflowsProps, State>
       return <div className="loading" />;
     }
     const isAdmin = this.props.user.isGroupAdmin();
+    const sortedRepos = (this.state.reposResponse?.repos ?? []).sort((a, b) => {
+      // Put repos with workflow history first.
+      const aHasHistory = this.renderActionList(a.repoUrl) != null;
```

Comment:
this is going to render the entire list once for every comparison it's involved in -- let's just make a map out of the repo urls in the history response before calling sort() and use that.

Reply:
Ah yeah - made a mental note to fix this but forgot to do it. Done